stable/anchore-engine/templates/ingress.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.ingress.enabled -}}
-apiVersion: networking.k8s.io/v1beta1
+apiVersion: {{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1beta1" -}} networking.k8s.io/v1beta1 {{- else }} extensions/v1beta1 {{- end }}
```

Comment:
This line has too much whitespace chomping and causes the following error:
```
Error: YAML parse error on anchore-engine/templates/ingress.yaml: error converting YAML to JSON: yaml: line 2: mapping values are not allowed in this context
```
Changing the conditional to look like this fixed the issue:
```
{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1beta1" }} 
```